AGILE-WOW.md

```diff
@@ -0,0 +1,102 @@
+# Agile Way of Working
+
+Agile is how we work at Fortytwo. In a nutshell, we take an iterative and incremental approach to our internal and customer deliveries -- based on the concepts of quality. transparency, flexibility and continuous improvement. 
+
+## Benefits of Agile
+
+* It allows Fortytwo´s high-performing and self-organizing teams to keep on learning and growing with time.
+
+* By keeping our customers and stakeholders in the loop and making changes according to their input, we deliver value and ensure that our products and services are truly in line with their requirements.
+
+* We significantly reduce the time to go-to-market.
+
+* We have better control over our work due to increased visibility. Working Agile allows us to better visualize the work we are doing. This also makes predicting risks and mitigating them much easier.
+
+* Agile allows Fortytwo teams to work in a flexible and self-managing way. We have increased autonomy and authority over our decisions through frequent and open collaboration. 
+
+* The Agile framework also produces important metrics like lead time, cycle time and throughput that helps teams measure their own performance, identify bottlenecks, overcome pitfalls like scope creep, and gives them the ability to make data-driven decisions to correct them.
+
+---
+
+## Basic Fortytwo Agile Guidelines
+
+We don't have a one specific or hard-set Agile way of working at Fortytwo. What we have is a common understanding of and respect for our inherent differences, which means we all work and interact differently -- and that's more than ALRIGHT! 
```

Comment:
Alright is usually used as an expression, which might stem from _all right_, but I would rather have the full set of words in a text like this.